TestingLowerBounds/ForMathlib/EReal.lean

```diff
@@ -2,16 +2,29 @@ import Mathlib.Data.Real.EReal
 import Mathlib.MeasureTheory.Constructions.BorelSpace.Order
 import Mathlib.MeasureTheory.Constructions.BorelSpace.Real
 
-open scoped ENNReal NNReal
+open scoped ENNReal NNReal Topology
+open Filter
 
 namespace EReal
 
 instance : CharZero EReal := inferInstanceAs (CharZero (WithBot (WithTop ℝ)))
 
+lemma eq_coe_of_ne_top_of_ne_bot {x : EReal} (hx : x ≠ ⊤) (hx' : x ≠ ⊥) : ∃ r : ℝ, x = r := by
+  induction x <;> tauto
```

Comment:
I don't know where you use this, but in most cases you should be able to replace it with 
`lift x to ℝ using ⟨hx, hx'⟩` (or something similar; I am writing this from memory). So that lemma is probably not needed?